poly/src/polynomial/univariate/sparse.rs

```diff
@@ -233,6 +233,26 @@ impl<F: Field> SparsePolynomial<F> {
     }
 
     /// Constructs a new polynomial from a list of coefficients.
+    /// Since the function does not automatically combine like terms, the input *has to be* in the
+    /// correct form: no multiple monomials of the same degree.
```

Comment:
```suggestion
    /// The function does not combine like terms and so multiple monomials
    /// of the same degree are ignored.
```

Reply:
I have updated the description according to your suggestions.

Reply:
I agree that having a negative example in the docs does feel weird.